frontend/app/routes/public/renew/$id/adult-child/update-home-address.tsx

```diff
@@ -546,12 +563,22 @@ function AddressInvalidDialogContent({ invalidAddress }: AddressInvalidDialogCon
       </div>
       <DialogFooter>
         <DialogClose asChild>
-          <Button id="dialog.address-invalid-close-button" startIcon={faChevronLeft} variant="alternative" size="sm">
+          <Button id="dialog.address-invalid-close-button" startIcon={faChevronLeft} variant="alternative" size="sm" data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult_Child:Dialog Cancel - Home address click">
```

Comment:
The button text actually says "Back":

```suggestion
          <Button id="dialog.address-invalid-close-button" startIcon={faChevronLeft} variant="alternative" size="sm" data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult_Child:Dialog Back - Home address click">
```

Reply:
Fixed text